packages/sui-ssr/server/dynamicRendering/index.js

```diff
@@ -46,7 +49,7 @@ export default function dynamicRendering(fallback, dynamicsURLS = []) {
           )
         )
     } else {
-      indexHTMLStream = fs.createReadStream(INDEX_HTML_PATH)
+      indexHTMLStream = fs.createReadStream(indexHtmlPath(req))
```

Comment:
not related to the PR change but, are we reading the file on each request? wouldn't that hurt performance wise?

Reply:
We are doing that because after closing the pipe, the next client will get a closed pipe. But you are right. We must move this from a Pipe to an inmemory var.

Reply:
Do you mean `indexHTMLStream` or `indexHtmlPath` variable?

Reply:
`indexHTMLStream `